VERSION

```diff
@@ -0,0 +1,6 @@
+# The base collectd version we are using (this is not a branch).
```

Comment:
I'm not sure what the parenthesis is telling me. Is the intention to say, "this doesn't have to match the git branch"?

Reply:
More strongly, "this doesn't specify a git branch at all". I've updated the comment to make this clearer.